wormhole-connect/src/config/routes.ts

```diff
@@ -107,4 +107,12 @@ export const RoutesConfig: {
     icon: XLabsIcon,
     pendingMessage: 'Waiting for Wormhole network consensus . . .',
   },
+  [Route.Mayan]: {
+    route: Route.Mayan,
+    name: 'Mayan Swap',
+    providedBy: 'Mayan Finance',
+    link: 'https://mayan.finance/',
+    icon: XLabsIcon,
```

Comment:
Should we change this to `WormholeIcon` since xlabs doesn't relay for mayan?

Reply:
this was a TODO that I never did because the new UI doesn't even use these icons